Hey @ArthurZucker, I tried to run `make fixup` but encountered an error in `repo-consistency`. 

```
Traceback (most recent call last):
  File "/home/users/user/temp/transformers/utils/update_metadata.py", line 337, in <module>
    check_pipeline_tags()
  File "/home/users/user/temp/transformers/utils/update_metadata.py", line 316, in check_pipeline_tags
    model = model[0]
IndexError: tuple index out of range
make: *** [Makefile:44: repo-consistency] Error 1
```

Anyway I thought I should not be encountering this in the first place since I only modified docstrings in a single file. So, I ran `black src/transformers/generation/logits_process.py` which gave:
```
All done! ✨ 🍰 ✨
1 file left unchanged.
```

Even `ruff src/transformers/generation/logits_process.py --fix` does nothing. Not sure what I should do to pass the failing test? I'm using `black==23.9.1` and `ruff==0.0.292` if that helps.

--- a/src/transformers/generation/logits_process.py
+++ b/src/transformers/generation/logits_process.py
@@ -282,8 +282,8 @@ class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
 
     Args:
         penalty (`float`):
-            The parameter for repetition penalty. 1.0 means no penalty. Above 1.0 penalizes previously generated tokens.
-            Between 0.0 and 1.0 rewards previously generated tokens. See [this 
+            The parameter for repetition penalty. 1.0 means no penalty. Above 1.0 penalizes previously generated
+            tokens. Between 0.0 and 1.0 rewards previously generated tokens. See [this
             paper](https://arxiv.org/pdf/1909.05858.pdf) for more details.
 
     Examples:

